.github/ISSUE_TEMPLATE/test-failure.yml

```diff
@@ -0,0 +1,51 @@
+name: Test Failure
+description: A test failure in CI
+labels: [">test-failure"]
+body:
+  - type: markdown
+    attributes:
+      value: Please fill out the following information, and ensure you have attempted to reproduce locally
+  - type: input
+    id: build_scan
+    attributes:
+      label: Build scan
+      description: A link to a Gradle buildscan with the failure
+    validations:
+      required: true
```

Comment:
Not all failures necessarily produce a scan. Maybe we should reword this to optionally be a link to a Jenkins job in those scenarios where things exploded prior to a scan being created.

Reply:
I reworded this to allow a jenkins job link, and relabeled it to "CI Link".